x/ccv/provider/types/keys.go

```diff
@@ -187,6 +187,10 @@ const (
 	// minimum power required to be in the top N per consumer chain.
 	MinimumPowerInTopNBytePrefix
 
+	// LastProviderConsensusValsPrefix is byte prefix for storing the last validator set
+	// sent to the consensus engine of the provider chain
+	LastProviderConsensusValsPrefix
```

Comment:
Can you please add this new prefix to `getAllKeyPrefixes()` in `x/ccv/provider/types/key_test.go`?

Reply:
Thanks!

Reply:
Thanks for the catch, done in https://github.com/cosmos/interchain-security/pull/1990/commits/bfe43c2f123c756c5234f43129e6c2e77540c4d8